lib/src/appengine_internal.dart

```diff
@@ -237,15 +240,49 @@ Future<db.DatastoreDB> _obtainDatastoreService(
 ///
 /// The returned [memcache.Memcache] will be usable within the current service
 /// scope.
-Future<memcache.Memcache> _obtainMemcacheInstance() async {
-  var rawMemcache;
-  try {
-    const int defaultMemcachedPort = 11211;
-    const String testKey = '__memcached_test_key';
-    final testValue = 'fobar-${new DateTime.now()}';
+Future<memcache.Memcache> _obtainMemcacheInstance(Logging logging) async {
+  //
+  // Try to connect to the app engine instance
+  //
+  const hostKey = 'GAE_MEMCACHE_HOST';
+  const portKey = 'GAE_MEMCACHE_PORT';
+
+  var hostVar = Platform.environment[hostKey];
+  var hostPort = Platform.environment[portKey];
+
+  if (hostVar != null && hostPort != null) {
+    var portNumber = int.parse(hostPort);
+    var instance = await _tryMemcacheInstance(hostVar, portNumber, logging);
+    if (instance != null) {
+      return instance;
+    }
+  }
+
+  //
+  // Try to connect to local host
+  //
```

Comment:
Remove this comment since it adds no value.

Reply:
yup